README.md

```diff
@@ -63,6 +63,7 @@ alternatives tend to be poorly-received**.
 * Submit a pull request. As a pull request the RFC will receive design
 feedback from the larger community, and the author should be prepared
 to revise it in response.
+* Find a champion on the ember-cli core team. Without a champion, the RFC will not progress.
```

Comment:
> Today, this is implicit I would like to make it explicit. Nothing lands unless someone on the core-team is able to represent the RFCs interests at the meetings. Today this ends up happening quite ad-hoc, by formalizing this I hope to empower more successful RFCs.

Agree 💯  . My two cents: the RFC format mention the ember-cli champion's name as well :) . It would help others reading the RFC to know and if the RFC gets disowned anyone else can easily pick up by pairing with the champion to get a knowledge transfer.